<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SARATR</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_O"/>
   <addaction name="actionPrint_S"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionFit_Screen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionPrint_S"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_Q"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionNormal_Size"/>
    <addaction name="actionFit_Screen"/>
    <addaction name="separator"/>
    <addaction name="actionROI"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSaliency">
    <property name="title">
     <string>Saliency(&amp;S)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menuSaliency"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <action name="actionOpen_O">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/imgs/open.png</normaloff>:/res/imgs/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionPrint_S">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/imgs/print.png</normaloff>:/res/imgs/print.png</iconset>
   </property>
   <property name="text">
    <string>Print(&amp;P)</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="text">
    <string>Quit(&amp;Q)</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/imgs/zoom-in.png</normaloff>:/res/imgs/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In (25%)</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/imgs/zoom-out.png</normaloff>:/res/imgs/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out (25%)</string>
   </property>
  </action>
  <action name="actionFit_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/imgs/zoom.png</normaloff>:/res/imgs/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Fit Screen</string>
   </property>
  </action>
  <action name="actionNormal_Size">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionROI">
   <property name="text">
    <string>ROI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
